src/main/java/luffy/Todo.java

```diff
@@ -0,0 +1,29 @@
+package luffy;
+
+/**
+ * Represents a task that has no start time, end time nor deadline
+ */
+public class Todo extends Task {
+
+    public Todo(String taskInfo, boolean isDone) {
+        super(taskInfo, isDone);
+    }
+
+    @Override
+    public String stringIsDone() {
+        if (checkIsDone()) {
+            return String.format("[T][X] %s", super.getTaskInfo());
+        } else {
+            return String.format("[T][ ] %s", super.getTaskInfo());
+        }
+    }
+
+    @Override
+    public String dataToSave() {
+        if (checkIsDone()) {
+            return String.format("TO-DO         | 1 | %s", super.getTaskInfo());
```

Comment:
I like how you made the string format easy to see in the data file by making the divider aligned for various tasks.